### PR TITLE
Python Docs Codebyte Project

### LINKED ISSUE #2820: [Edit] Python Add Codebytes
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Please add Codebytes to the listed entries.
https://www.codecademy.com/resources/docs/python/files/rmdir
https://www.codecademy.com/resources/docs/python/files/seek

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) when working on the PR for this issue.

Bear in mind the following two principles:
1) There should only be one Codebyte per entry listed as `## Codebyte Example` at the end of the entry.
2) The codebytes should be new examples, not copies of the prior examples.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Robot011011**
### Description

Added Codebyte Examples to the listed entries:
https://www.codecademy.com/resources/docs/python/files/rmdir
https://www.codecademy.com/resources/docs/python/files/seek

Closes #2820 

### Type of Change

- Editing an existing entry (added codebyte examples)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/files/terms/rmdir/rmdir.md [hunk 1]
```diff
@@ -50,3 +50,27 @@ os.rmdir("myfolder")
 # Delete folder in another directory
 os.rmdir("/path/to/myfolder")
 ```
+
+## Codebyte Example
+
+In this example the `./project` directory is deleted using the `.rmdir()` method:
+
+```codebyte/python
+# Imports os module
```
Comment (caupolicandiaz) [inline content/python/concepts/files/terms/rmdir/rmdir.md:59]: I would remove the import os comment because it's not really adding any information

Comment (caupolicandiaz) [reply content/python/concepts/files/terms/rmdir/rmdir.md:59]: remove on both entries

**Review by caupolicandiaz — APPROVED** [review]
@Robot011011 Overall, this looks good, I've just added one comment. Please update the entries. This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@Robot011011 Looks good.

**Pull request merged**